projects/sport-data-valley/manual_seeds/training_responses.rb

```diff
@@ -0,0 +1,248 @@
+# frozen_string_literal: true
+require_relative 'helpers/people_helper'
+srand(123)
+
+START_DATE = Time.zone.parse('2021-08-01').beginning_of_day
```

Comment:
I would prefer relative dates in these kind of scripts, so that you can call them a year later and still have recent responses generated without any editing of the file.